browser/src/Services/Configuration/IConfigurationValues.ts

```diff
@@ -86,7 +86,7 @@ export interface IConfigurationValues {
     // By default, user's init.vim is not loaded, to avoid conflicts.
     // Set this to `true` to enable loading of init.vim.
     // Set this to a string to override the init.vim path.
-    "oni.loadInitVim": string | boolean
+    // "oni.loadInitVim": string | boolean
```

Comment:
Was gonna leave a note since its a big change essentially with a default config setting as false its currently not possible to determine if the user's config has set the value to `false` or if `oni's` default did. The only way I found to actually tell the difference was to have either an explicit user setting or no setting. I guess if each option had info re where it was set that could help.

As it stands if the default remains then basically if the user has their config set to false then there's no way to check so the notification always appears if the setting is false since theres a false value but no distinction

Reply:
Ah I see, I'd prefer the check to be explicit - leaving the functionality of `oni.loadInitVim` as-is. IMO, it's not easy to explain `undefined` states of config values to users - so I'd prefer to be explicit, and use a separate explicit setting to check if we've shown the init.vim - prompt - like `prompt.hasShownInitVimMessage`.